### ISSUE TITLE
Set time more than 100 days

### ISSUE THREAD

**Issue by itsmefurzy** [anugotta/FlipTImerView#5]
Hello,

I need to enhance this library into more than 100 days, is it possible?

**Comment by anugotta** [comment]
wow! Never thought about this scenario:) will definitely keep in pipeline. Thanks

**Comment by itsmefurzy** [comment]
Yeah, i didn't get another great library for countdown timer except this. Many thanks if you give this update.

**Comment by javadghane** [comment]
@anugotta 
Hi, thanks for this great library.
Is this issue on the update list?
